Paper.tex

```diff
@@ -1335,7 +1335,7 @@ \section{Modified Merkle Patricia Tree}\label{app:trie}
 \end{cases}
 \end{equation}
 
-In a manner similar to a radix tree, when the trie is traversed from root to leaf, one may build a single key-value pair. The key is accumulated through the traversal, acquiring a single nibble from each branch node (just as with a radix tree). Unlike a radix tree, in the case of multiple keys shared the same prefix or in the case of a single key having a unique suffix, two optimising nodes are provided. Thus while traversing, one may potentially acquire multiple nibbles from each of the other two node types, extension and leaf. There are three kinds of nodes in the trie:
+In a manner similar to a radix tree, when the trie is traversed from root to leaf, one may build a single key-value pair. The key is accumulated through the traversal, acquiring a single nibble from each branch node (just as with a radix tree). Unlike a radix tree, in the case of multiple keys sharing the same prefix or in the case of a single key having a unique suffix, two optimising nodes are provided. Thus while traversing, one may potentially acquire multiple nibbles from each of the other two node types, extension and leaf. There are three kinds of nodes in the trie:
```

Comment:
Here I changed: *"in the case of multiple keys __shared__ the same prefix or in the case of a single key having a unique suffix"* into *"in the case of multiple keys __sharing__ the same prefix or in the case of a single key having a unique suffix"*